sdk/keyvault/azure-keyvault-keys/tests/test_examples_crypto_async.py

```diff
@@ -107,6 +107,6 @@ class TestCryptoExamples(AsyncKeyVaultTestCase):
         from azure.keyvault.keys.crypto import SignatureAlgorithm
 
         verified = await client.verify(SignatureAlgorithm.rs256, digest, signature)
-        assert verified.result is True
+        assert verified.is_valid is True
```

Comment:
My bad, style point, should be `assert verified.is_valid` because this evaluates to e.g. `assert True is True`

Reply:
switched, thanks!